packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -329,6 +327,9 @@ export default mixins(
 			closeCredentialNewDialog () {
 				this.credentialNewDialogVisible = false;
 			},
+			openTagManager() {
+				this.$store.dispatch('ui/openTagsManagerModal');
+			},
```

Comment:
ya these could be mutations. I went back on forth on these. the reason why I decided on `dispatch` is because it's easiest to decouple from state. it also allow multiple mutations. so I am mostly optimizing for flexibility in the future to make changes.